recipes/netcdf/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import CMake, ConanFile, tools
+import glob
+import os
+
+
+class NetcdfConan(ConanFile):
+    name = "netcdf"
+    description = "The Unidata network Common Data Form (netCDF) is an interface for scientific data access and a freely-distributed software library that provides an implementation of the interface."
+    topics = ("unidata", "unidata-netcdf", "networking")
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/Unidata/netcdf-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "netcdf4": [True, False],
+        "with_hdf5": [True, False],
+        "cdf5": [True, False],
+        "dap": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "netcdf4": True,
+        "with_hdf5": True,
+        "cdf5": True,
+        "dap": True,
+    }
+    generators = "cmake_find_package", "cmake_find_package_multi", "cmake"
```

Comment:
I rarely see both `cmake_find_package` and `cmake_find_package_multi` are needed